src/classes/ACCT_IndividualAccounts_TEST.cls

```diff
@@ -1010,8 +1010,8 @@ private class ACCT_IndividualAccounts_TEST {
     
 /* TESTING PHONE NUMBER WORKFLOW -------------------------------------------------------
  
-    /// <name> createContactWithPhoneNoPref </name>
-    /// <summary> Test Method for creating a contact with a phone number and no preferences set. </summary>
+    /*********************************************************************************************************
+    * @description Test Method for creating a contact with a phone number and no preferences set. </summary>
```

Comment:
Bingo - left the methods commented out, and they don't get put in the ApexDoc, but I changed the syntax so that if they ever get restored, they're ready to go.